### PR TITLE
Fizz buzz checker

### PR DESCRIPTION
**Pull request by dhruv-mahajan**
PR to merge our Fizz Buzz Generator program to master

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example.project</groupId>
+    <artifactId>git-demo-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+<dependencies>
```
Comment (vishalsinha21) [inline pom.xml:10]: indentation missing

File: src/main/java/dummyPack/FizzBuzz.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        }else{
+            System.out.println(number);
+        }
+
+        System.out.println(result);
+    }
+
+    public static String getFizzBuzzOutput(int number){
+
+        if(number%15==0){
```
Comment (vishalsinha21) [inline src/main/java/dummyPack/FizzBuzz.java:25]: auto formatting missing

File: src/test/java/dummyPack/FizzBuzzTest.java [hunk 3]
```diff
@@ -6,8 +6,21 @@
...
+        String actual=FizzBuzz.getFizzBuzzOutput(9);
+        assertEquals(expected,actual);
+    }
+
+    @Test
+    public void test_if_Buzz_is_returned_For_testValue(){
+        String expected="Buzz";
+        String actual=FizzBuzz.getFizzBuzzOutput(20);
+        assertEquals(expected,actual);
+    }
```
Comment (shivangi8293) [inline src/test/java/dummyPack/FizzBuzzTest.java:25]: Test case for no matching case number is missing

File: src/main/java/dummyPack/FizzBuzz.java [hunk 4]
```diff
@@ -6,21 +6,31 @@
...
             result = getFizzBuzzOutput(number);
         }else{
-            System.out.println("Invalid Number!! Try again.");
+            System.out.println(number);
         }
 
         System.out.println(result);
     }
 
     public static String getFizzBuzzOutput(int number){
```
Comment (shivangi8293) [inline src/main/java/dummyPack/FizzBuzz.java:23]: method can be refactored to reduce if else

File: src/test/java/dummyPack/FizzBuzzTest.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
+package dummyPack;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+public class FizzBuzzTest {
+    @Test
+    public void test_if_FizzBuzz_is_returned_For_testValue(){
```
Comment (vishalsinha21) [inline src/test/java/dummyPack/FizzBuzzTest.java:8]: should_return_FizzBuzz_if_number_is_divisible_by_15

**Pull request closed without merging**